static/app/components/events/viewHierarchy/wireframe.tsx

```diff
@@ -29,6 +36,8 @@ function Wireframe({hierarchy, selectedNode, onNodeSelect}: WireframeProps) {
   const theme = useTheme();
   const [canvasRef, setCanvasRef] = useState<HTMLCanvasElement | null>(null);
   const [overlayRef, setOverlayRef] = useState<HTMLCanvasElement | null>(null);
+  const [zoomIn, setZoomIn] = useState<HTMLButtonElement | null>(null);
```

Comment:
why do you need both of these or why do you need them at all?

Reply:
Yeah, it is pretty weird, but I'm doing it because I'm dependent on some state in the closure of the `useEffect` hook right now where I'm adding event listeners for the canvas elements, so I'm also assigning the click handlers for these buttons there as well.

I'm going to see if I can tweeze these apart in a separate PR because I wanted to implement that model idea and making more of this logic testable. I think these tasks will be pretty related